ravenframework/Models/PostProcessors/Validations/PhysicsGuidedCoverageMapping.py

```diff
@@ -399,6 +413,12 @@ def pcmTdep(featData, msrData, targData):
 
     pcmVersion = self.pcmType
 
+
+    if self.ReconstructionError != None:
+      recError = self.ReconstructionError #reconstruction error to determine the rank of time series data.
+    else:
+      recError = None
+
```

Comment:
There is a default for self.ReconstructionError, so you do not need the if else check list.